### PR TITLE
Fix/did document controller api

### PR DESCRIPTION
**Pull request by mustafasalfiti**
Seperating the api documentation from the DidDocumentController.java into a seperate file 
#145

### CODE REVIEW

**Comment by borisrizov-zf** [comment]
@mustafasalfiti please rebase your local branch using the upstream/develop

**Comment by mustafasalfiti** [comment]
done

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                                    """)
+                    }) }),
+            @ApiResponse(responseCode = "404", description = "Wallet not found with provided bpn", content = {
+                    @Content(examples = {
+                            @ExampleObject(name = "Wallet not found with provided bpn", value = """
+                                    {
+                                      "type": "about:blank",
+                                      "title": "Wallet not found for BPN BPNL00000000000",
+                                      "status": 404,
+                                      "detail": "Wallet not found for BPN BPNL00000000000",
```
Comment (borisrizov-zf) [inline src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java:42]:
```suggestion
                                      "detail": "Wallet not found for identifier BPNL00000000000",
```

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                                      }
+                                    }
+                                    """)
+                    }) }),
+            @ApiResponse(responseCode = "404", description = "Wallet not found with provided bpn", content = {
+                    @Content(examples = {
+                            @ExampleObject(name = "Wallet not found with provided bpn", value = """
+                                    {
+                                      "type": "about:blank",
+                                      "title": "Wallet not found for BPN BPNL00000000000",
```
Comment (borisrizov-zf) [inline src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java:102]:
```suggestion
                                      "title": "Wallet not found for identifier BPNL00000000000",
```

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                                      "status": 404,
+                                      "detail": "Wallet not found for BPN BPNL00000000000",
```
Comment (borisrizov-zf) [inline src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java:104]:
```suggestion
                                      "detail": "Wallet not found for identifier BPNL00000000000",
```

**Review by borisrizov-zf — CHANGES_REQUESTED** [review]

File: src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java (hunk 3 above)
Comment (mustafasalfiti) [reply src/main/java/org/eclipse/tractusx/managedidentitywallets/apidocs/DidDocumentControllerApiDocs.java]: Bpn changed to identifier

**Review by borisrizov-zf — APPROVED** [review]

**Comment by borisrizov-zf** [comment]
@mustafasalfiti hi, you'll have to rebase again as we've pulled changes in with #176. Do this for all your branches - that will fix the pipeline.

**Pull request merged**